elixir/apps/domain/lib/domain/config/definitions.ex

```diff
@@ -454,7 +454,7 @@ defmodule Domain.Config.Definitions do
   defconfig(
     :clients_upstream_dns,
     {:json_array, {:embed, Domain.Config.Configuration.ClientsUpstreamDNS},
-     validate_unique: true},
+     validate_unique: false},
```

Comment:
Just making sure I'm following -- this is for validating the IP portion only right? Or would this allow dupes of `1.1.1.1:853`?

Reply:
Ah you wrote a custom uniqueness validator, I see

Reply:
Yeah, I had to write a custom uniqueness validator (or at least I didn't see an easy way to pass a custom uniqueness validator function to this definition), because I needed to normalize all of the addresses.  Then I needed to turn off this `validate_unique`, because I would then get 2 messages when there was a duplication of the exact same string.

Reply:
👍